.vscode/cspell.json

```diff
@@ -262,6 +262,10 @@
       "filename": "sdk/maps/maps-search/review/maps-search.api.md",
       "words": ["bbox", "Neighbourhood", "Xstr"]
     },
+    {
+      "filename": "sdk/maps/maps-route-rest/review/maps-route.api.md",
```

Comment:
@ckairen I wonder if there is a way to ignore all folders suffixed with `-rest` from cspell. Rest clients don't do any renames on the service properties and names and are autogenerated.